web/crux/src/app/agent/agent.service.ts

```diff
@@ -136,7 +136,7 @@ export default class AgentService {
         this.configService,
         nodeId,
         this.jwtService.sign(token),
-        new Date(now + AgentService.SCRIPT_EXPIRATION),
+        new Date(now + JWT_EXPIRATION),
```

Comment:
it's not the same thing imho

Reply:
The script's expiration is the same as the JWT expiration. I can revert it back, but I renamed it to make it more specific.